fairlearn/widget/fairness_dashboard.py

```diff
@@ -28,26 +28,28 @@
 
 
 class FairlearnDashboard(object):
-    """The dashboard class, wraps the dashboard component.
+    r"""The dashboard class, wraps the dashboard component.
 
-    :param sensitive_features: A matrix of feature vector examples (# examples x # features),
+    Parameters
+    ----------
+    sensitive_features : {:class:`numpy.ndarray`, list[][], :class:`pandas.DataFrame`, :class:`pandas.Series`}  # noqa: E501
+        A matrix of feature vector examples (# examples x # features),
         these can be from the initial dataset, or reserved from training.
-    :type sensitive_features: numpy.ndarray or list[][] or pandas.DataFrame or pandas.Series
-    :param y_true: The true labels or values for the provided dataset.
-    :type y_true: numpy.ndarray or list[]
-    :param y_pred: Array of output predictions from models to be evaluated. Can be a single
+    y_true : {:class:`numpy.ndarray`, list[]}
+        The true labels or values for the provided dataset.
+    y_pred : {:class:`numpy.ndarray`, list[][], list[], dict {string: list[]}}
```

Comment:
I don't think you need :class: here - only in the main run of the text.

Reply:
It does render differently. If I don't put this then only one of the types gets the link, the others are just text.

Reply:
i'm not sure if those are the right type descriptors, also lists of types seem to work fine in metrics: https://fairlearn.github.io/master/api_reference/fairlearn.metrics.html

Reply:
I'll just go back to what we had before. Not going to waste much time on this, since this will be deleted shortly. We should get it figured out in the new repository, though.